setup.py

```diff
@@ -59,6 +59,7 @@ def create_version_file():
     'psutil>=5.0.0',
     'scikit-learn>=0.20.0,<0.23',
     'networkx>=2.3,<3.0',
+    'openml'
```

Comment:
Why is this dependency added? If it is only for tests, can it moved out of the setup.py dependencies so people not interested in tests can avoid installing it?

Reply:
It is needed in a tutorial: docs/tutorials/course/mlp.md.

Do you know of a way to only have it as a dependence for this tutorial?

This tutorial (done by Thibaut) is actually really good. It will also be used in the CVPR tutorial.

Reply:
@zhanghang1989 Would this live in test_requirements, or does the CI use something else?

Reply:
If it is only needed during the docs build, we can add it here https://github.com/awslabs/autogluon/blob/master/docs/build_contrib.yml

Reply:
@mseeger Can you move the openml dependency to the build_contril.yml?